CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Run `cargo-deny` in CI. #4101
 - Check provider database with CI. #4099 
 - Switch to DEFERRED transactions #4100
+- Compress `mime_headers` column with HTML emails stored in database
```

Comment:
Not adding a PR number, it can be easily found as it has the same name. Maybe we can add a CI check that a changelog entry is a substring of the PR name?